### PR TITLE
Global stylings and issues fixing

### LINKED ISSUE #203: Improvement: add responsiveness to the app
**Is your feature request related to a problem? Please describe.**

Currently the app is not completely responsive, some pages cannot be viewed on mobile or smaller screens. We want to add this improvement and make this app mobile responsive.

**Describe the solution you'd like**

- [ ] All the pages and components should be mobile responsive
- [ ] Should fit most of the common mobile screen dimensions
- [ ] Component sizes are proportional to the screen size

### Checklist:

- [x] I have double checked that the issue does not currently exist. 
- [x] I have added the appropriate labels (e.g. feature, front-end, back-end).
- [ ] I have added the **Platform Project** project

### LINKED ISSUE #204: Global styling fixing
**Is your feature request related to a problem? Please describe.**

Currently stylings across the app are not consistent, a global styling fixing should be carried out to make sure the app looks aesthetic and professional.

**Describe the solution you'd like**

Fix stylings for the following page and all the related components:
- [x] Fix stylings for landing page
- [x] Fix stylings for manage account page
- [x] Fix stylings for dashboard
- [x] Fix stylings for bill splitting page
- [x] Fix stylings for manage flat page
- [x] Fix stylings for manage account page
- [x] Fix stylings for notes page
- [x] Fix stylings for maintenance issues page
- [x] Fix stylings for announcement page
- [x] Fix stylings for chores (tasks) page

### Checklist:

- [x] I have double checked that the issue does not currently exist. 
- [x] I have added the appropriate labels (e.g. feature, front-end, back-end).
- [ ] I have added the **Platform Project** project

### PR DESCRIPTION
**Pull request by rainasong**
# Description

Fixed global stylings and numerous other bugs and issues along the way.

- Fixed stylings for pages:
  - Landing page
  - Manage account page
  - Dashboard
  - Bill splitting page
  - Manage flat page
  - Manage account page
  - Notes page
  - Maintenance issues page
  - Announcement page
  - Chores (tasks) page

- Fixed stylings for components:
  - CreateHouseModal
  - JoinHouseModal
  - Navigation
  - UserDropdown
  - BackButton
  - AnnouncementCard
  - AnnouncementModalForm
  - DashboardAnnouncement
  - NotesGrid
  - newNoteModal
  - plainModal
  - secretModal
  - NewBillCard
  - NewIssueCard
  - Footer
  - OccupantForm
  - HouseSettings

- Added custom default font
- Added new default avatar
- Temporarily hidden firebase emulator warning UI (for demo purpose)
- Removed announcement page force refresh
- Fixed NewBillCard and NewIssueCard text fields not automatically clean up after 'done' is clicked issue
- Added mobile responsiveness to all pages

Fixes/resolves #203 and #204 

## Demos
Landing page, login and dashboard

https://user-images.githubusercontent.com/37087904/161404131-90f46776-2c42-4f07-8df9-5a7ff4413adc.mp4

Bill splitting page

https://user-images.githubusercontent.com/37087904/161404141-4de18e1d-aaec-45c3-abbb-23cd34dd7055.mp4

Tasks page

https://user-images.githubusercontent.com/37087904/161404145-7b4fce1e-d865-495b-9358-2b4e3e28e22c.mp4

Manage flat page

https://user-images.githubusercontent.com/37087904/161404158-3e7cac80-0791-48de-9ee4-af1582b58679.mp4

Notes page

https://user-images.githubusercontent.com/37087904/161404163-2952d7c7-b37b-49c7-bafa-3ce42dc0df5b.mp4

Maintenance issues page

https://user-images.githubusercontent.com/37087904/161404183-f3b19ebb-0e54-4f3f-bf0a-94bba5a00db9.mp4

Announcement page

https://user-images.githubusercontent.com/37087904/161404189-b03010c0-a1fc-40e6-bff1-ede56c93b293.mp4

Mobile run through

https://user-images.githubusercontent.com/37087904/161404198-13a28051-287d-4cae-add7-72e26ffbc377.mp4

## Type of change

Please delete options that are not relevant.

- [x] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **New feature** (non-breaking change which adds functionality)
- [x] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

Leave blank if not applicable

I have completed these steps when making this pull request:

- [ ] I have assigned my name to the issue
- [ ] I have moved the issue to the **In Progress** column
- [ ] I have labelled the PR appropriately
- [ ] I have assigned myself to the PR

Before opening the PR for review:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added attributions to new dependencies and resources
- [ ] I have moved the linked issue to the **Review in Progress** column

### CODE REVIEW

**Review by noahjade — DISMISSED** [review]
You are an absolute legend 🥇 🌹  Thank you for putting this massive amount of effort in. Your attention to detail is unparalleled!

**Review by jimwang6012 — DISMISSED** [review]
LGTM, thanks for doing the global style fix!

File: packages/frontend/src/components/manage/HouseSettings.tsx [hunk 1]
```diff
@@ -76,11 +68,12 @@ const HouseSettings: React.FC<HouseSettingsProps> = (props) => {
               });
             }}
             color="secondary"
-            width="60%"
+            // className='w-full'
```
Comment (butterplup) [inline packages/frontend/src/components/manage/HouseSettings.tsx]: Not sure if it's important but did you want to leave in this commented out code `className='w-full'`? Same for the other instances of it.

File: packages/frontend/src/components/notes/note/newNoteModal.tsx [hunk 2]
```diff
@@ -118,7 +118,6 @@ const NewNoteModal: React.FC<NewNoteModalProps> = ({
         <Text size={'1.25rem'} margin="1.5%">
           Type
         </Text>
-        {/* This is just as a start. Will use Tailwind for this and not some ugly select tag haha */}
```
Comment (butterplup) [inline packages/frontend/src/components/notes/note/newNoteModal.tsx:121]: This comment is funny, it really reminds me of that reading we did on working with legacy code. Temporary fixes are never temporary.

**Review by butterplup — DISMISSED** [review]
You've done a fantastic job here, this looks great!

**Review by hajineats — DISMISSED** [review]
mindblown

File: packages/frontend/src/components/manage/HouseSettings.tsx (hunk 1 above)
Comment (rainasong) [reply packages/frontend/src/components/manage/HouseSettings.tsx]: Nice catch! Thanks for letting me know, I was going to delete them but there were too many files haha

**Review by jimwang6012 — APPROVED** [review]
LGTM, thanks for doing the global style fix!

**Review by noahjade — APPROVED** [review]

**Review by lin8231 — APPROVED** [review]
Amazing work! Thanks so much Raina

**Review by butterplup — APPROVED** [review]
Amazing stuff, LGTM 👍

**Pull request merged**